test/support/chr/list_pagination.rb

```diff
@@ -9,12 +10,12 @@ def pagination(factory_name, class_name, list_of_modules)
       select_last_module_from_list(list_of_modules)
       while @loaded_items < class_name.count do
         sleep(2.0)
-        assert page.has_css?('div.item-title', count: @loaded_items)
+        assert page.has_css?('div.item-title', count: @loaded_items), "Number of elements on page don't mach number of elements what must be loaded: Present: #{page.all('div.item-title').count}, Must be: #{@loaded_items}"
```

Comment:
Line is too long. [221/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.